scene/main/scene_tree.h

```diff
@@ -190,6 +191,9 @@ class SceneTree : public MainLoop {
 	Node *prev_scene = nullptr;
 	Node *pending_new_scene = nullptr;
 
+	// Initialized lazily and destroyed eagerly to decrease RAM usage, since it contains a lot of text.
+	LicensesDialog *licenses_dialog = nullptr;
```

Comment:
I'm not sure I like having this in SceneTree, I wonder if we can find a better place to host it. Not sure where that would be though, I need to think more about it.